src/swapd/runtime.rs

```diff
@@ -1286,7 +424,7 @@ impl Runtime {
                     BusMsg::Sync(SyncMsg::Task(watch_height_bitcoin)),
                 )?;
 
-                trace!("Watch height monero");
+                self.log_trace("Watch height monero");
```

Comment:
General comment about these log trace, isn't the trace level too low? IMO these messages could be at the debug level and trace is only used when printing array of bytes, big hex strings, etc.

Reply:
Resolved by async discussion with @TheCharlatan